atlas/AtlasMl/tests/test_pipelines_unit.py

```
@@ -0,0 +1,106 @@
+import pytest
+import numpy as np
+from unittest.mock import MagicMock, patch, call
```

Ruff (0.11.9)

`unittest.mock.call` imported but unused

Remove unused import: `unittest.mock.call`

(F401)